crates/ruff_python_parser/tests/snapshots/invalid_syntax@statements__match__invalid_lhs_or_rhs_pattern.py.snap

```diff
@@ -0,0 +1,1178 @@
+
+   |
+ 6 |     case _ + 3j:
+ 7 |         pass
+ 8 |     case (1 | 2) + 4j:
+   |                       ^ Syntax Error: Expected an expression
+ 9 |         pass
+10 |     case [1, 2] + 5j:
+11 |         pass
+   |
+
+
+   |
+ 7 |         pass
+ 8 |     case (1 | 2) + 4j:
+ 9 |         pass
+   | ^^^^^^^^ Syntax Error: unexpected indentation
+10 |     case [1, 2] + 5j:
+11 |         pass
+   |
+
+
+   |
+ 8 |     case (1 | 2) + 4j:
+ 9 |         pass
+10 |     case [1, 2] + 5j:
+   |      Syntax Error: Expected a statement
+11 |         pass
+12 |     case {True: 1} + 6j:
+   |
+
+
+   |
+ 8 |     case (1 | 2) + 4j:
+ 9 |         pass
+10 |     case [1, 2] + 5j:
+   |          ^^^^^^^^^^^ Syntax Error: invalid assignment target
+11 |         pass
+12 |     case {True: 1} + 6j:
+   |
+
+
+   |
+ 8 |     case (1 | 2) + 4j:
+ 9 |         pass
+10 |     case [1, 2] + 5j:
+   |                      ^ Syntax Error: Expected an expression
+11 |         pass
+12 |     case {True: 1} + 6j:
+13 |         pass
+   |
+
+
+   |
+ 9 |         pass
+10 |     case [1, 2] + 5j:
+11 |         pass
+   | ^^^^^^^^ Syntax Error: unexpected indentation
+12 |     case {True: 1} + 6j:
+13 |         pass
+   |
+
+
+   |
+10 |     case [1, 2] + 5j:
+11 |         pass
+12 |     case {True: 1} + 6j:
+   |      Syntax Error: Expected a statement
+13 |         pass
+14 |     case 1j + 2j:
+   |
+
+
+   |
+10 |     case [1, 2] + 5j:
+11 |         pass
+12 |     case {True: 1} + 6j:
+   |          ^^^^^^^^^^^^^^ Syntax Error: invalid assignment target
+13 |         pass
+14 |     case 1j + 2j:
+   |
+
+
+   |
+10 |     case [1, 2] + 5j:
+11 |         pass
+12 |     case {True: 1} + 6j:
+   |                         ^ Syntax Error: Expected an expression
+13 |         pass
+14 |     case 1j + 2j:
+15 |         pass
+   |
+
+
+   |
+11 |         pass
+12 |     case {True: 1} + 6j:
+13 |         pass
+   | ^^^^^^^^ Syntax Error: unexpected indentation
+14 |     case 1j + 2j:
+15 |         pass
+   |
+
+
+   |
+12 |     case {True: 1} + 6j:
+13 |         pass
+14 |     case 1j + 2j:
+   |      Syntax Error: Expected a statement
+15 |         pass
+   |
+
+
+   |
+12 |     case {True: 1} + 6j:
+13 |         pass
+14 |     case 1j + 2j:
+   |          ^^^^^^^ Syntax Error: invalid assignment target
+15 |         pass
+   |
+
+
+   |
+12 |     case {True: 1} + 6j:
+13 |         pass
+14 |     case 1j + 2j:
+   |                  ^ Syntax Error: Expected an expression
+15 |         pass
+16 | 
+17 | match invalid_rhs_pattern:
+   |
+
+
+   |
+13 |         pass
+14 |     case 1j + 2j:
+15 |         pass
+   | ^^^^^^^^ Syntax Error: unexpected indentation
+16 | 
+17 | match invalid_rhs_pattern:
+   |
+
+
+   |
+15 |         pass
+16 | 
+17 | match invalid_rhs_pattern:
+   |  Syntax Error: Expected a statement
+18 |     case 1 + Foo():
+19 |         pass
+   |
+
+
+   |
+17 | match invalid_rhs_pattern:
+18 |     case 1 + Foo():
+   |              ^^^^^ Syntax Error: invalid rhs pattern
```

Comment:
I understand.

> Does this match the literal_expr grammar?

No, I borrowed it from the existing code.

> Does it only allow numbers after a `+`?

Only complex literal is allowed after a `+` or `-` because it can only occur in a complex literal pattern. I'll update the error message to better reflect that.

I'll update the LHS / RHS terminology in a follow-up PR.